### PR TITLE
Fixed link second frame of tour

### PR DESCRIPTION
**Pull request by nenadev21**
- Fixed link second frame of tour
- Deleted unused file imports
- Added continuos flow on impact page

### CODE REVIEW

File: src/components/Pages/ActionsPage/ActionsPage.js [hunk 1]
```diff
@@ -205,10 +205,24 @@ class ActionsPage extends React.Component {
...
+        disableOverlayClose: true,
+      },
+      // ...
+    ];
+
     return (
       <>
         <ProductTour
-          steps={this.state.steps}
+          steps={steps}
```
Comment (BradHN1) [inline src/components/Pages/ActionsPage/ActionsPage.js:225]: So I think we can take steps out of the state variables?

File: src/components/Pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -129,10 +129,56 @@ class HomePage extends React.Component {
...
+        hideFooter: true,
+      },
+    ];
+
     return (
-      <div className="boxed_wrapper">
+      <>
         <ProductTour
-          steps={this.state.steps}
+          steps={steps}
```
Comment (BradHN1) [inline src/components/Pages/HomePage/HomePage.js:181]: same as the other file, we can remove the setState call

File: src/components/Pages/ActionsPage/ActionsPage.js (hunk 1 above)
Comment (nenadev21) [reply src/components/Pages/ActionsPage/ActionsPage.js:225]: yes. I think that'll work

File: src/components/Pages/HomePage/HomePage.js (hunk 2 above)
Comment (nenadev21) [reply src/components/Pages/HomePage/HomePage.js:181]: yes. I think that'll work

**Review by BradHN1 — APPROVED** [review]
I made the minor fixes

**Pull request merged**